cypress/e2e/tests/pages/manager/drivers.spec.ts

```diff
@@ -50,7 +50,7 @@ describe('Drivers', { testIsolation: 'off', tags: ['@manager', '@adminUser'] },
     });
 
     it('can deactivate a cluster driver', () => {
-      RkeDriversPagePo.navTo();
+      driversPage.goTo();
```

Comment:
Would you be able to expand on why the navTo -> goTo changes midway through a test (here and others) are required? I'm concerned these are genuine bugs (and in theory we shouldn't need to reload)

Reply:
I think it's a timing issue; another case of automation being too fast. I've removed the goTo's and added `.waitForPage()` along with the navTo's and it seems to have helped.